docs/docs/30-how-to-guides/15-working-with-freight.md

```diff
@@ -112,12 +112,22 @@ resource as it progresses through the `Stage`s of a pipeline.
 
 This is conveniently accomplished via the Kargo CLI:
 
-```shell
-kargo update freight \
-  f5f87aa23c9e97f43eb83dd63768ee41f5ba3766 \
-  --alias frozen-tauntaun \
-  --project kargo-demo
-```
+1. **Using Name**
+   ```shell
+   kargo update freight \
+     --project=kargo-demo \
+     --name=f5f87aa23c9e97f43eb83dd63768ee41f5ba3766 \
+     --new-alias=frozen-tauntaun
+   ```
+
+1. **Using Old Alias**:
+   If you have the old alias and prefer to use it
+   ```shell
+   kargo update freight \
+     --project=kargo-demo \
+     --old-alias=mortal-dragonfly \
+     --new-alias=frozen-tauntaun
+   ```
```

Comment:
As a general principle, we should stay away from ordered lists when order doesn't matter.

```suggestion
```shell
kargo update freight \
  --project=kargo-demo \
  --name=f5f87aa23c9e97f43eb83dd63768ee41f5ba3766 \
  --new-alias=frozen-tauntaun
```

Alternatively, the `Freight` to which you wish to assign a new alias can be referenced by its existing alias:

```shell
kargo update freight \
  --project=kargo-demo \
  --old-alias=mortal-dragonfly \
  --new-alias=frozen-tauntaun
```
```